stix_shifter_modules/trustar/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,251 @@
+{
+  "IP": {
+    "key": "ipv4-addr.value",
+    "object": "ipv4-addr"
+  },
+  "MALWARE": {
+    "key": "x-trustar-malware.value",
+    "object": "x-trustar-malware"
+  },
+  "URL": {
+    "key": "url.value",
+    "object": "url"
+  },
+  "CVE": {
+    "key": "vulnerability.id",
```

Comment:
there's no `vulnerability` standard stix object. it needs to go into a custom object

Reply:
Done the changes